tests/api/test_api.py

```diff
@@ -17,7 +18,7 @@
 from erica.infrastructure.sqlalchemy.repositories.base_repository import EntityNotFoundError
 from tests.application.job_service.test_job_service import MockEricaRequestRepository, MockRequestController, MockDto, \
     PickableMock
-from tests.utils import create_unlock_code_request, \
+from tests.utils import create_send_grundsteuer, create_unlock_code_request, \
```

Comment:
In the test_api, please add also the grundsteuer use case for the following parameterized tests:

- test_if_get_job_returns_failure_status
- test_if_get_job_returns_not_found